librz/util/vector.c

```diff
@@ -236,6 +236,50 @@ RZ_API void *rz_vector_flush(RzVector *vec) {
 	return r;
 }
 
+// CLRS Quicksort. It is slow, but simple.
+#define VEC_INDEX(a, i) (char *)a + elem_size *(i)
+static void vector_quick_sort(void *a, size_t elem_size, size_t len, RzPVectorComparator cmp, bool reverse) {
+	rz_return_if_fail(a);
+	if (len <= 1) {
+		return;
+	}
+	size_t i = rand() % len, j = 0;
+	void *t, *pivot;
+
+	t = (void *)malloc(elem_size);
+	pivot = (void *)malloc(elem_size);
+	if (!t || !pivot) {
+		RZ_FREE(t);
```

Comment:
Use simple free, no need to set them to NULL since we immediately return afterwards.